internal/engine/wazevo/backend/isa/amd64/machine.go

```diff
@@ -687,7 +687,13 @@ func (m *machine) lowerXmmRmR(instr *ssa.Instruction) {
 
 	// rn is being overwritten, so we first copy its value to a temp register,
 	// in case it is referenced again later.
-	tmp := m.copyToTmp(rn.r)
+	tmp := m.c.AllocateVReg(x.Type())
+	if rn.r != regalloc.VRegInvalid {
+		m.insert(m.allocateInstr().asMovRR(rn.r, tmp, _64))
+	} else {
+		tmp = m.c.AllocateVReg(x.Type())
+		m.insert(m.allocateInstr().asLEA(rn.amode, tmp))
```

Comment:
this doesn't make sense to me

Reply:
my mistake, I was looking at DIV and didn't pay enough attention here. Would it be acceptable to `panic("TODO")` for now? If I am not wrong, it should be ok to use movd/movss, but then there's a test case missing.

Reply:
well, can we just kill this PR before getting swamped with div/rem stuff?